### PR TITLE
feat: update screenshots for catalog

### PR DESCRIPTION
**Pull request by danielgolden**
Update screenshots to fit catalog requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,11 @@
...
 
-![Screenshot #1](assets/screenshots/screenshot_01.png)
+![Screenshot #1](assets/screenshots/nr1-workload-geoops-1.png)
 <p align="center">
-<img src="assets/screenshots/screenshot_02.png" width="32.5%">
-<img src="assets/screenshots/screenshot_03.png" width="32.5%">
-<img src="assets/screenshots/screenshot_04.png" width="32.5%">
+<img src="assets/screenshots/nr1-workload-geoops-2.png" width="32.5%">
+<img src="assets/screenshots/nr1-workload-geoops-3.png" width="32.5%">
+<img src="assets/screenshots/nr1-workload-geoops-4.png" width="32.5%">
```
Comment (jbeveland27) [inline README.md:25]: Did you mean to update these? The updated screenshots are under `catalog/screenshots/`.

Comment (danielgolden) [reply README.md:25]: 🤦

Comment (danielgolden) [reply README.md:25]: Going to undo this change

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 Workloads GeoOps provides a geographic exploration of [Workload](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-one-workloads-isolate-resolve-incidents-faster) data (collections of Entities) tied to geographic data, including location metadata. Find additional information in the [docs](docs/documentation.md) directory.
 
-![Screenshot #1](assets/screenshots/screenshot_01.png)
+![Screenshot #1](assets/screenshots/nr1-workload-geoops-1.png)
```
Comment (jbeveland27) [inline README.md:21]: Just missing this last one:

```suggestion
![Screenshot #1](assets/screenshots/screenshot_01.png)
```

**Pull request merged**